generator/generator.go

```diff
@@ -1350,19 +1360,9 @@ func (g *generator) writeWhitespace(indentLevel int, n parser.Whitespace) (err e
 }
 
 func (g *generator) writeText(indentLevel int, n parser.Text) (err error) {
-	vn := g.createVariableName()
-	// vn := sExpr
-	if _, err = g.w.WriteIndent(indentLevel, vn+" := "+createGoString(n.Value)+"\n"); err != nil {
-		return err
-	}
-	// _, templ_7745c5c3_Err = templ_7745c5c3_Buffer.WriteString(vn)
-	if _, err = g.w.WriteIndent(indentLevel, "_, templ_7745c5c3_Err = templ_7745c5c3_Buffer.WriteString("+vn+")\n"); err != nil {
-		return err
-	}
-	if err = g.writeErrorHandler(indentLevel); err != nil {
-		return err
-	}
-	return nil
+	quoted := strconv.Quote(n.Value)
```

Comment:
🥇 Nicely done. Makes me wonder whether `createGoString` should die... Can be done in a separate refactor though.

Reply:
Looks like it could be part of the string literals. I will take a look at this in a different PR.